site/content/en/latest/contributions/design/envoy-extension-policy.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "EnvoyExtensionPolicy "
+---
+
+## Overview
+
+This design document introduces the `EnvoyExtensionPolicy` API allowing system administrators to configure traffic
```

Comment:
can we incorporate some of these below points in this doc

* there are now 2 kinds of extensibility today
  * extending EG API by configuring xDS directly using EnvoyPatchPolicy & ExtensionManager
  * the other kind, highlighted in this doc that extends and adds newer functionality on top of what Envoy offers
  
Also a note around how users have been using  EnvoyPatchPolicy & ExtensionManager to directly configure this extensions, but this API provides a safer and more scoped API